src/test/java/com/hocs/test/glue/foi/SoftCloseStepDefs.java

```diff
@@ -0,0 +1,7 @@
+package com.hocs.test.glue.foi;
+
+import com.hocs.test.pages.decs.BasePage;
+
+public class SoftCloseStepDefs extends BasePage {
+
```

Comment:
Yeah this is more of a placeholder since the only thing to do at Soft Close is to reopen the case and I've used the 'And I click the "Reopen" button' step instead of writing up a different one

Reply:
understood